fv3gfs-physics/fv3gfs/physics/stencils/get_phi_fv3.py

```diff
@@ -1,6 +1,7 @@
 from gt4py.gtscript import BACKWARD, PARALLEL, computation, interval
 
-from fv3gfs.physics.global_config import *
+from fv3gfs.physics.global_config import FIELD_FLT
```

Comment:
Probably not about this PR but what is a FIELD_FLT and how is it different from a FloatField?

Reply:
There's no difference actually. Let me clean this up.